src/app/components/sing-in/sing-in.component.html

```diff
@@ -1,19 +1,76 @@
-<div class="sing-in-form">
-  <h3>Sing In Form</h3>
-  <div>
-    <h5>Email</h5>
-    <span class="p-float-label">
-      <input id="float-input" type="text" pInputText [(ngModel)]="email" />
-    </span>
+<form class="sing-in-form" [formGroup]="loginForm">
+  <h2 class="mb-5">{{ title }}</h2>
+  <div class="field flex flex-column align-items-center">
+    <div>
+      <label for="email" class="block mb-2">Email</label>
+      <span class="p-float-label">
+        <input
+          class="border-round-sm"
+          type="email"
+          pInputText
+          placeholder="enter email"
+          formControlName="emailInput"
+        />
+      </span>
+    </div>
+    <div class="mt-1">
+      <small
+        class="p-error block"
+        *ngIf="
+          loginForm.controls['emailInput'].dirty &&
+          loginForm.hasError('required', 'emailInput')
+        "
+      >
+        *Email is require</small
+      >
+    </div>
+    <div class="mt-1">
+      <small
+        class="p-error block"
+        *ngIf="loginForm.hasError('email', 'emailInput')"
+      >
+        *Email must be like vasil@gmail.com</small
+      >
+    </div>
   </div>
-  <div>
-    <h5>Password</h5>
-    <p-password [feedback]="false"></p-password>
+  <div class="field flex flex-column align-items-center">
+    <div>
+      <label for="password" class="block mb-2">Password</label>
+      <span class="p-float-label">
+        <input
+          class="border-round-sm"
+          type="password"
+          pInputText
+          placeholder="enter password"
+          formControlName="password"
+        />
+      </span>
+    </div>
+    <div class="mt-1">
+      <small
+        class="p-error block"
+        *ngIf="
+          loginForm.controls['password'].dirty &&
+          loginForm.hasError('required', 'password')
+        "
+      >
+        *Password is require</small
+      >
+    </div>
+    <div class="mt-1">
+      <small
+        class="p-error block"
+        *ngIf="loginForm.hasError('minlength', 'password')"
+      >
+        *Name must be at least 6 characters long</small
+      >
+    </div>
   </div>
-  <div class="mt-3 mb-3">
+  <div class="mt-2 mb-3">
     <p-button
-      (click)="onBtnClick()"
-      (click)="close()"
+      (onClick)="onBtnClick()"
```

Comment:
Try to use function names that make more sense. As a function does something it is a good idea to put a verb in its name that describes what exactly the function does.

Reply:
Fixed